cli/command/image/list.go

```diff
@@ -59,6 +61,10 @@ func NewImagesCommand(dockerCLI command.Cli) *cobra.Command {
 	flags.StringVar(&options.format, "format", "", flagsHelper.FormatHelp)
 	flags.VarP(&options.filter, "filter", "f", "Filter output based on conditions provided")
 
+	flags.BoolVar(&options.tree, "tree", false, "List multi-platform images tree (EXPERIMENTAL)")
+	flags.SetAnnotation("tree", "api", []string{"1.47"})
```

Comment:
I think this should be `version` instead of `api`, e.g. see https://github.com/docker/cli/blob/211a5403b3671787c62db68cde3208f6fa95661a/cli/command/image/build.go#L145

Reply:
OH right, good catch!